GiTinder/Controllers/GitAsyncRequest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using GiTinder.Data;
+using GiTinder.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace GiTinder.Controllers
+{
+    [Route("test")]
+    [ApiController]
+    public class GitAsyncRequest
+    {
+        private readonly GiTinderContext _context;
+
+      
+        [HttpGet("username")]
```

Comment:
@Riceqrisp this endpoint path should have a variable or a parameter for the username

Reply:
changed the base